thunderfish/peakdetection.py

```diff
@@ -2,6 +2,84 @@
 import numpy as np
 
 
+def std_threshold(data, samplerate, win_shift, std_factor=5.):
+    """
```

Comment:
as helper functions the *_threshold functions() should go below the detect_peaks() functions (right before the trim() functions)